CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
-## 1.6.0-RC2
+## 1.6.0-RC3
```

Comment:
Please don't overwrite the RC2 section. Add RC3 (or Unreleased) and your change above it.